apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/offerbook/BisqEasyOfferbookView.java

```diff
@@ -72,7 +72,7 @@ public final class BisqEasyOfferbookView extends ChatView<BisqEasyOfferbookView,
     private DropdownTitleMenuItem atLeastTitle;
     private CheckBox hideUserMessagesCheckbox;
     private Label channelHeaderIcon, marketPrice, removeWithOffersFilter, removeFavouritesFilter;
-    private HBox appliedFiltersSection, withOffersDisplayHint, onlyFavouritesDisplayHint;
+    private HBox appliedFiltersSection, withOffersDisplayHint, onlyFavouritesDisplayHint, expiredMessagesIndicator;
```

Comment:
Why is this only added to the offerbook and not in general to all the chats?

Reply:
Note that for private chat messages we do not have an expiry date as its not gossip messages, thus this is not needed. But for common chat messages it should be applied.